auth-web/src/components/auth/manage-business/AddNameRequestForm.vue

```diff
@@ -173,7 +187,7 @@ export default class AddNameRequestForm extends Vue {
     }
   }
 
-  private resetForm (event: string): void {
+  resetForm (event: string): void {
```

Comment:
We don't need the private public protected accessors within a vue component. It's just extra words that clutter the code.